doc/sources/oneapi-gpu.rst

```diff
@@ -107,3 +107,7 @@ An example on how to patch your code with Intel CPU/GPU optimizations:
                [8., 7.], [8., 8.], [25., 80.]], dtype=np.float32)
    with config_context(target_offload="gpu:0"):
       clustering = DBSCAN(eps=3, min_samples=2).fit(X)
+
+Note: current offloading behavior restricts fitting and inference of any model to be
+in the same context or absence of context. For example, model trained in context with
+target_offload="gpu:0" will throw error if inference is done outside of context.
```

Comment:
```suggestion
Note: current offloading behavior restricts fitting and inference of any model to be
in the same context or absence of context. For example, model trained in GPU context with
target_offload="gpu:0" will throw error if inference is done in CPU context.
```

Reply:
It's applicable for any context currently